interfaces/builtin/opengl.go

```diff
@@ -117,6 +117,10 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 # Imagination PowerVR driver
 /dev/pvr_sync rw,
 
+# ARM Mali driver
+/dev/mali[0-9]* rw,
+/dev/dma_buf_te rw,
```

Comment:
Please also extend the unit test for checking these (or at least one of them).

Reply:
@mardy 
Do you mean https://github.com/snapcore/snapd/blob/master/interfaces/builtin/opengl_test.go#L83?
It seems to me that the unit test for mail can be covered by L83 & L84.

Reply:
Yes, I would add at least one line checking for `/dev/mali[0-9]*`

Reply:
@mardy I have added a commit for it.
Here is the test result.
$ ./run-checks
...
All good, what could possibly go wrong.

btw... for the first commit, I have built a test snapd and verified it on the device with Mali. It worked.
Here is the detailed information: https://bugs.launchpad.net/baoshan/+bug/1958487

Reply:
Thanks!